components/_footer.scss

```diff
@@ -12,3 +19,34 @@ footer {
     float: none;
   }
 }
+
+.ethos-rant {
+  color: $tws-white;
+
+  @include media($tablet) {
+    @include span-columns(5 of 10);
+  }
+}
+
+.admin-links {
+  font-family: $gibson;
+  font-weight: 600;
+  text-align: center;
```

Comment:
Properties should be ordered font-family, font-weight, margin-bottom, text-align